lib/access_token_agent/connector.rb

```diff
@@ -10,20 +12,32 @@ def initialize(host:,
       @client_id = client_id
       @client_secret = client_secret
       @fake_auth = fake_auth
+      set_default_config!
```

Comment:
I think it should also be possible to set the accces_token_path with the normal configuration options instead of allowing a change only after the connector is instantiated.